app/src/main/res/menu/alarms_menu.xml

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
:warning: Notification icons are specific to the build flavor. Please do not delete them.

Reply:
I know, but there should already be a Vector Version of that Image :thinking:

Reply:
If I understand you correctly you are referring to [drawable-anydpi-v24](https://github.com/EventFahrplan/EventFahrplan/tree/master/app/src/ccc36c3/res/drawable-anydpi-v24). The `v24` suffix of the folder looks like the vector drawable might only be supported as of SDK level 24. For now the icon assistant in Android Studio still generates PNGs, too. I guess there is a reason for it. But if you find out that PNGs are no longer necessary for old and new Android versions that would be beautiful.

Reply:
Please note the general (not specific to notification icons) SDK level hints on these pages:

- https://developer.android.com/develop/ui/views/graphics/vector-drawable-resources
- https://developer.android.com/develop/ui/views/graphics/drawables
- https://developer.android.com/reference/android/graphics/drawable/VectorDrawable
- https://developer.android.com/jetpack/androidx/releases/vectordrawable/

There is also a couple of videos from Google Developers and conferences about the topic.

Reply:
Also found a [thread on Stackoverflow](https://stackoverflow.com/questions/34016107/can-we-use-vectordrawable-or-vectorxml-as-icons-for-push-notifications-in-androi) about that topic now, I'll add those images back in :+1: